### ISSUE TITLE
How to finetune with custom dataset

### ISSUE THREAD

**Issue by nahidalam** [ZHKKKe/MODNet#169]
Did anyone ever tried to finetune the MODNet model with custom dataset? Wondering if you have any tutorial.

**Comment by denddyprod** [comment]
> Did anyone ever tried to finetune the MODNet model with custom dataset? Wondering if you have any tutorial.

I tried, but I have problems with training script. The loss is not decreasing.

Maybe someone will help us with a good training script or with some clear ideas about finetuning. Thanks!

**Comment by nahidalam** [comment]
Can you share it @denddyprod ? It is in some github maybe I can collaborate there?

**Comment by saidulu31** [comment]
> Can you share it @denddyprod ? It is in some github maybe I can collaborate there?

I'm interested in fine-tuning. Could you explain the process

**Comment by zzzcyyyw** [comment]
> > Did anyone ever tried to finetune the MODNet model with custom dataset? Wondering if you have any tutorial.
> 
> I tried, but I have problems with training script. The loss is not decreasing.
> 
> Maybe someone will help us with a good training script or with some clear ideas about finetuning. Thanks!

May I ask if you have solved the problem of losses not going down? I recently had the same problem and was struggling with it.
